src/integration-test/java/uk/gov/ida/verifylocalmatchingserviceexample/rules/TestDatabaseRule.java

```diff
@@ -32,12 +33,14 @@ protected void after() {
     }
 
     private void setUpDatabase() {
-        handle.execute("create table verifiedPid(pid TEXT)");
+        Flyway flyway = new Flyway();
+        flyway.setDataSource(this.appRule.getConfiguration().getDataSourceFactory().getUrl(), this.appRule.getConfiguration().getDataSourceFactory().getUser(), this.appRule.getConfiguration().getDataSourceFactory().getPassword());
+        flyway.migrate();
     }
 
     public void ensurePidExist(String verifiedPid) {
         handle.begin();
-        handle.execute("insert into verifiedPid values('" + verifiedPid + "')");
+        handle.execute("insert into verifiedPid (pid, user) values('" + verifiedPid + "', 1)");
```

Comment:
I'm not particularly a fan of this always being against the same user, but it seems the easiest way. We could fetch a record from the database and use the id of that, but that seems overly complex to understand at a glance. Selecting a random integer (knowing that the migrations introduce users with ids 1-25) would leave a chance of tests intermittently failing if the migration is ever edited (which it ought not to be, but still - it might, or one might be added which deletes one of the users for some reason). At least if there is no user with id 1 this will always fail and it will be clear why.